src/index.html

```diff
@@ -17,6 +17,43 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <h1></h1>
+    <header>
+      <a
+        href="#"
+        class="logo"
+      >
+        <img
+          src="images/logo.png"
+          alt="Logo"
+        />
+      </a>
+      <nav>
+        <ul>
+          <li>
+            <a
+              href="#"
+              class="is-active"
+            >
+              Apple
+            </a>
+          </li>
+          <li><a href="#">Samsung</a></li>
```

Comment:
Remember about correct indentation between parent and child elements. Each list item should have 2-space indentation from its parent `ul`.